features_and_tools/pubsub.rst

```diff
@@ -0,0 +1,563 @@
+              "version": "59f47104-395a-4883-9689-259651939ca2"
+            },
+            "resource": "{\n  \"resourceType\": \"Patient\",\n  \"id\": \"example-id\",\n  \"meta\": {\n    \"versionId\": \"59f47104-395a-4883-9689-259651939ca2\",\n    \"lastUpdated\": \"2023-10-26T15:39:44.319+00:00\"\n  }\n}",
+            "changeType": "create"
+          }
+        ]
+      },
+      ...
+    }
+
+    
+  **Metadata**
+
+  * ``messageType`` - always ``urn:message:Firely.Server.Contracts.Messages.V1:ResourcesChangedEvent``
+  * ``headers.fhir-release`` specifies the FHIR version, either ``STU3``, ``R4``, or ``R5``
+
+  **Message body**
+
+  * ``reference`` - A reference to the resource for which the change is communicated
+  * ``resource`` - A flattened json of the resource reflecting its state after the change was made
+  * ``changeType`` - The kind of change that was made, either a ``create``, ``update``, or ``delete``
+
+
+ResourcesChangedLightEvent
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If enabled, Firely Server can also publish ``ResourcesChangedLightEvent`` messages. This message type will contain information on the resource change but will not include the entire resource resource body. As it is with the ``ResourcesChangedEvent``, clients can subscribe to the corresponding message type ``ResourcesChangedLightEvent``.
+
+.. attention::
+    This functionality is not yet supported for SQLite or MongoDB.
+
+.. note::
+  Publishing of this event is disabled by default and must be enabled in the :ref:`configuration<pubsub_configuration>`.
+
+.. container:: toggle
+
+  .. container:: header
+
+    Event
+
+  .. code-block::
+
+    {
+      "messageType": [
+        "urn:message:Firely.Server.Contracts.Messages.V1:ResourcesChangedLightEvent"
+      ],
+      "headers": {
+        "fhir-release": "R4"
+      },
+      "message": {
+        "changes": [
+          {
+            "reference": {
+              "resourceType": "Patient",
+              "resourceId": "fsiTestingPatient",
+              "version": "41098b04-68ce-4b04-bce2-2d3c738d24f7"
+            },
+            "changeType": "create"
+          }
+        ]
+      },
+      ...
+    }
+
+  **Metadata**
+
+  * ``messageType`` - always ``urn:message:Firely.Server.Contracts.Messages.V1:ResourcesChangedLightEvent``
+  * ``headers.fhir-release`` specifies the FHIR version, either ``STU3``, ``R4``, or ``R5``
+
+  **Message body**
+
+  * ``reference`` - A reference to the resource for which the change is communicated
+  * ``changeType`` - The kind of change that was made, either a ``create``, ``update``, or ``delete``
+
+
+Database tracking initialization
+-----------------------
+
+.. _pubsub_sql_tracking_init:
+
+SQL Server
+^^^^^^^^^^
+
+If you want to enable publishing notifications whenever resources get changed in Firely Server and you use SQL Server, some initial configuration is required. This can be done automatically by Firely Server or manually.
```

Comment:
Maybe good to add what this initial configuration entails and why it is necessary.